File: VB/E2894.Web/SampleReport.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI

Namespace E2894.Web
	Partial Public Class SampleReport
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()

			customParameter.Type = GetType(GangOfFour)
			customParameter.Value = GangOfFour.None
		End Sub
	End Class
End Namespace

File: CS/E2894.Web/SampleReport.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;

namespace E2894.Web {
    public partial class SampleReport : DevExpress.XtraReports.UI.XtraReport {
        public SampleReport() {
            InitializeComponent();

            customParameter.Type = typeof(GangOfFour);
            customParameter.Value = GangOfFour.None;
        }
    }
}
